decidim-core/lib/decidim/form_builder.rb

```diff
@@ -202,7 +202,6 @@ def editor(name, options = {})
           toolbar: toolbar,
           disabled: options[:disabled]
         }.merge(editor_images_options(options)), style: "height: #{lines}rem")
-        template += error_for(name, options) if error?(name)
```

Comment:
This change wasn't in the original backport. Is it intentional?

Reply:
Nop, fixing now.